compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToLLVM.cpp

```diff
@@ -309,6 +309,19 @@ class ConvertIREEBindingSubspanOp : public ConvertToLLVMPattern {
     llvmFuncOp.setArgAttr(llvmBufferArg.getArgNumber(),
                           LLVM::LLVMDialect::getAlignAttrName(),
                           rewriter.getI32IntegerAttr(16));
+    // The ranges used within the bindings are guaranteed not to alias
```

Comment:
Style: comments get proper punctuation here and below (add .'s)
(https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar)